vonk/releasenotes/releasenotes.rst

```diff
@@ -17,6 +17,53 @@ Upgrading Vonk
 
 See :ref:`upgrade` for information on how to upgrade to a new version of Vonk.
 
+.. _vonk_releasenotes_350:
+
+Release 3.5.0
+-------------
+
+Feature
+^^^^^^^
+
+#. Search reference by identifier: FHIR R4 allows you to `search a reference by its identifier <http://hl7.org/fhir/R4/search.html#reference>`_. We added support for this in Vonk. Note that any identifiers in reference elements have to be indexed by Vonk. For new data that is done automatically. But if you want to use this on existing data, you have to :ref:`reindex for the searchparameters <feature_customsp_reindex_specific>` you want to use it on. E.g. Observation.patient. 
+#. AuditEvent logging: In release 3.3.0 we already added support for logging audit information to a file. With this release we add to that logging that same information in AuditEvent resources. These resources are written to the Vonk Data database (not the Administration database). Users are not allowed to update or delete these resources. See :ref:`auditing` for more background.
+#. Audit logging: we added ``[Request]`` or ``[Response]`` to the log lines so you can distinguish them better.
+#. Sort! We started implementing :ref:`sorting <restful_search_sort>`. This release provides sorting for searchparameters of the types string, number, uri, reference, datetime and token, on the repositories SQL, SQLite and Memory. On the roadmap is extending this support to MongoDb and to quantity searchparameters.
```

Comment:
Change '!' behind Sort to ':', or was this intentional?
Also add a link to _restful_search_sort.

Reply:
The ! was intentional because we finally did this. But : fits better with the rest.
There *is* already a link to restful_search_sort, or did you mean something else?

Reply:
No, I overlooked the link, sorry!